### ISSUE TITLE
doesn't handle self.assertDictContainsSubset

### ISSUE THREAD

**Issue by rtdean** [pytest-dev/unittest2pytest#22]
If you have a unittest that contains an `assertDictContainsSubset`, unittest2pytest will happily ignore/skip over it.

```python
class ExampleTest(unittest.TestCase):
    def test_dict_subset():
        subset = {'a': 1}
        superset = {'a': 1, 'b': 2}
        self.assertDictContainsSubset(subset, superset)
```

This can be replaced with the following:
```python
def test_dict_subset():
    subset = {'a': 1}
    superset = {'a': 1, 'b': 2}
    assert dict(superset, **subset) == superset
```

**Comment by htgoebel** [comment]
Thanks for sharing this idea! Looks easy to implement.

**Comment by RonnyPfannschmidt** [comment]
@htgoebel as far as i can tell the checked in test reverses the order

**Comment by htgoebel** [comment]
Fixed in 758fa33.
